src/tracing/BUILD.gn

```diff
@@ -21,6 +21,14 @@ config("tracing_enabled") {
   defines = [ "MATTER_TRACING_ENABLED" ]
 }
 
+config("tracing_multiplexed") {
+  defines = [ "MATTER_TRACE_MULTIPLEXED" ]
```

Comment:
It is be preferable to generate a header with buildconfig_header to facilitate crossing build systems.

Reply:
Done.